forem/forem
# Fix GitHub Readme liquid tag relative URLs

## What type of PR is this?


## Description

This is a bug fix for the github README liquid tag.

The relative links contained in the README's HTML are replaced by absolute links.

These absolute links have broken URL's: https://raw.githubusercontent.com/forem/forem/main/#what-is-forem

This PR aims to fix the links with a correct URL: https://github.com/forem/forem#what-is-forem

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

 - Click the "Create Post" button to start writing a new post
 - In the content of the post add a github README liquid tag to a repo with relative links in the beginning of the README: `{% github nlepage/gophers %}`
 - Enter preview mode
 - Click one of the relative links to check if the URL is correct

## Added/updated tests?

      have not been included_
